app/src/main/java/com/xmartlabs/template/helper/GeneralErrorHelper.java

```diff
@@ -67,32 +67,21 @@ public GeneralErrorHelper() {
     BaseProjectApplication.getContext().inject(this);
   }
 
-  private void logCrashlyticsError(ServiceExceptionWithMessage exceptionWithMessage) {
+  private void logError(ServiceExceptionWithMessage exceptionWithMessage) {
     String url = ServiceHelper.getUrl(exceptionWithMessage.getResponse().raw());
     int resultCode = exceptionWithMessage.getCode();
     String headers = exceptionWithMessage.getResponse().headers().toString();
     String body = exceptionWithMessage.getErrorBody();
-    Crashlytics.setString(CRASHLYTICS_KEY_URL, url);
```

Comment:
Crashlytics manage this information in a different way, for example you can make filters based on the "tags". If you want, remove they form the message, but in my opinion the "tags" are important! Also, be careful, because some loggers like Sentry doesn't allow this extra information